cmd/bb_browser/browser_service.go

```diff
@@ -203,12 +203,28 @@ func (s *BrowserService) handleActionCommon(w http.ResponseWriter, req *http.Req
 
 		// TODO(edsch): Should we support Std{out,err}Raw as well? Buildbarn doesn't generate them.
 		var err error
-		actionInfo.StdoutInfo, err = s.getLogInfo(ctx, "Standard output", instance, actionResult.StdoutDigest)
+		if actionResult.StdoutDigest != nil {
+			actionInfo.StdoutInfo, err = s.getLogInfo(ctx, "Standard output", instance, actionResult.StdoutDigest)
```

Comment:
I think we can implement this with a bit less code by extending `getLogInfo()` to do the necessary checks for us. What are your thoughts on these changes?

https://github.com/buildbarn/bb-browser/compare/inline-logs

Reply:
That is indeed shorter! Let's merge it and close that PR.